### PR TITLE
NGSTACK-786 upgrade swiper 4 > 11

### PR DESCRIPTION
**Pull request by TTrakostanec**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
         "js-cookie": "^3.0.5",
         "photoswipe": "^5.3.4",
         "popper.js": "^1.15.0",
-        "swiper": "^4.5.0"
+        "swiper": "11.0.2"
```
Comment (emodric) [inline package.json:80]: Why not keep `^11.0.2` so we can benefit from newer 11.x releases?

**Comment by mlukac** [comment]
Changes all seem correct. Class names, different imports, and different swiper breakpoint logic.
Because of the delicate changes it would be best to test all swipers on some instance's /showcase page.

**Review by mlukac — APPROVED** [review]

**Review by emodric — APPROVED** [review]
@TTrakostanec Update the package.json to allow for later 11.x versions of Swiper, and this can be merged.

**Comment by TTrakostanec** [comment]
@emodric package.json updated

**Comment by emodric** [comment]
Thanks @TTrakostanec @mlukac !

**Pull request merged**